roles/confluent.kafka_broker/tasks/set_principal.yml

```diff
@@ -39,11 +67,18 @@
     - listener['sasl_protocol'] | default(sasl_protocol) | normalize_sasl_protocol == 'none'
     - listener['ssl_enabled'] | default(ssl_enabled) | bool
     - listener['ssl_mutual_auth_enabled'] | default(ssl_mutual_auth_enabled) | bool
+    - kafka_broker_final_properties['ssl.principal.mapping.rules'] is not defined
```

Comment:
i think we care about `ssl.principal.mapping.rules` as well as when that property is added to the end of the interbroker listener prefix (although that is unlikely to be set)

Reply:
I would need a little more details around this.